airbyte-integrations/connectors/source-northpass-lms/README.md

```diff
@@ -0,0 +1,103 @@
+# Northpass Lms Source
+
+This is the repository for the Northpass Lms configuration based source connector.
+For information about how to use this connector within Airbyte, see [the documentation](https://docs.airbyte.com/integrations/sources/northpass-lms).
+
+## Local development
+
+### Prerequisites
+
+* Python (`^3.10`)
+* Poetry (`^1.7`) - installation instructions [here](https://python-poetry.org/docs/#installation)
+
+
+
+### Installing the connector
+
+From this connector directory, run:
+```bash
+poetry install --with dev
+```
+
+
+### Create credentials
+
+**If you are a community contributor**, follow the instructions in the [documentation](https://docs.airbyte.com/integrations/sources/northpass-lms)
+to generate the necessary credentials. Then create a file `secrets/config.json` conforming to the `src/source_northpass_lms/spec.yaml` file.
+Note that any directory named `secrets` is gitignored across the entire Airbyte repo, so there is no danger of accidentally checking in sensitive information.
+See `sample_files/sample_config.json` for a sample config file.
+
+
+### Locally running the connector
+
+```
+poetry run source-northpass-lms spec
+poetry run source-northpass-lms check --config secrets/config.json
+poetry run source-northpass-lms discover --config secrets/config.json
+poetry run source-northpass-lms read --config secrets/config.json --catalog sample_files/configured_catalog.json
+```
+
+### Running tests
+
+To run tests locally, from the connector directory run:
+
+```
+poetry run pytest tests
+```
```

Comment:
```suggestion
```

Since there are no unit tests in the low-code connector, this block does not apply.